### ISSUE TITLE
Změna deploye na jinou větev (např. production)

### ISSUE THREAD

**Issue by lukaszach** [gugcz/web-admin#21]
Vzhledem k tomu, že **master** je středobod repository - běžně je to "_uzel_", do kterého proudí všechny MR, tak není vhodné, aby se ten vždy vydával na produkci. 

Vhodný způsob pro continuous integration je použití jiné větve, do které se **master** merguje a pomocí CI dojde teprve tehdy k releasu.

Navrhuji vytvořit novou větev **production**, do které se v případě potřeby bude mergovat **master**. Následně by došlo k releasu. 

Výhody:

- souběžný vývoj
- možnost prereleasu (testing/staging)
- stále aktuální master

Postup:

- [x] upravit aktuální CI na větev production
- [x] vytvořit větev production

-----------------

Současně:

- [x] v CI změnit npm za yarn

**Motivace**: _Aplikace neobsahuje ani yarn.lock, ani package-lock.json. Vhodné je zvolit jednu variantu (asi yarn.lock) jako součást repository a podle ní pak instalovat závislosti. Tím vzniká vyšší pravděpodobnost i jistota, že se něco při drobné úpravě nerozbije na závislostech._

**Comment by lukaszach** [comment]
@Horm dává ti to smysl?

**Comment by HormCodes** [comment]
@lukaszach jojo souhlas, předpokládám, že to takto provedeme i u ostatních repozitářů, že?

**Comment by lukaszach** [comment]
@Horm ano, zde bych asi začal, protože to je docela podstatné a pak zkusil jinde. Tím by se dalo vytvořit i testovací prostředí s vlastní URL - osobně nevím pricing za takovou další instanci. Dalo by se to domyslet.
